### PR TITLE
add LDAP_AUTH_SYNC_USER_RELATIONS setting

### PR DESCRIPTION
**Pull request by mnach**
to have a possibility to store additional ldap-data in database.

Created from maintainer recommendations in the comment of #14 PullRequest.

### CODE REVIEW

**Comment by etianen** [comment]
This looks good. I'll have a proper look on Monday when I'm back from
holiday.

**Comment by mnach** [comment]
Ok. Have a high time!

File: django_python3_ldap/utils.py [hunk 1]
```diff
@@ -30,6 +30,11 @@ def clean_user_data(user_data):
     return user_data
 
 
+def sync_user_relations(user, ldap_data):
+    # do nothing by default
+    return ldap_data
```
Comment (etianen) [inline django_python3_ldap/utils.py]: This should really just be ``pass``, since the return value is ignored by the calling function.

**Comment by etianen** [comment]
Looks great! If you could make the following minor tweaks, I'll pull.

- [ ] Add a note to the docs about the new setting.
- [ ] Make the default implementation of ``sync_user_relations`` simply ``pass``.

**Comment by etianen** [comment]
Hmm? Why closed?

**Pull request closed without merging**